common/types.ts

```diff
@@ -0,0 +1,107 @@
+export interface User {
+    email: string;
+    firstName: string;
+    lastName: string;
+    userId: number; 
+    gender: Gender;
+    school: string;
+    education: Education;
+    yearOfEducation: YearOfEducation;
+    ethnicities: Ethnicities;
+    shirtSize: ShirtSize;
+    applicationStatus: ApplicationStatus;
+    major: string;
+    minor: string;
+    resumeLink: string;
+    timeZone: string;
+    learningGoals: string;
+}
+
+export interface PortalState {
+    openDate: Date;
+    closeDate: Date;
+    maxAttendees: number;
+    numAttendees: number;
+    totalConfirmed: number;
+
+}
+
+enum Gender {
+    Nonbinary="Nonbinary", Female="Female", Male="Male", Unspecified="Unspecified"
+}
+
+enum Education {
+    HighSchool="HighSchool", Undergraduate="Undergraduate", Graduate="Graduate", 
+    Doctorate="Doctorate"
+}
+
+type YearOfEducation = 1 | 2 | 3 | 4 | 5 | "5+";
+
+enum Ethnicities {
+    "Indigenous/AlaskaNative"="Indigenous/AlaskaNative", Asian="Asian", "Black/AfricanAmerican"="Black/AfricanAmerican", 
```

Comment:
Why are these strings? Can we just do something like IndigenousAlaskaNative and just display the prettier way?

Reply:
Yeah we could do that – the string was so that we could have the / in the name haha. But yeah we can do that

Reply:
Yup I agree with Daniel the camel case would be easier to read